src_c/simd_blitters_sse2.c

```diff
@@ -490,21 +479,25 @@ alphablit_alpha_sse2_argb_no_surf_alpha_opaque_dst(SDL_BlitInfo *info)
                     /* src(ARGB) -> src1 (00000000ARGBARGB) */
                     LOAD_64_INTO_M128(srcp64, &src1);
 
-                    /* created squashed alpha -> mm_src_alpha
-                     * (0000000000000A0A) */
+                    /* isolate alpha channels
+                     * 00000000A1000A2000 -> mm_src_alpha */
+                    mm_src_alpha = _mm_andnot_si128(mm_rgb_mask, src1);
+
+                    /* shift right to position alpha channels for manipulation
+                     * 000000000A1000A200 -> mm_src_alpha*/
+                    mm_src_alpha = _mm_srli_si128(mm_src_alpha, 1);
+
```

Comment:
Might be helpful to have a comment indicating where the logic switched from 8 bit integers over to 16 bit
```suggestion
                    /* contents considered as 8 16bit integers :
                     * [00][00][00][00][0A1][00][0A2][00]
                     * [7 ][6 ][5 ][4 ][ 3 ][2 ][ 1 ][0 ]
```

Was something I made while following through the logic of the shuffle.